src/include/sof/math/log.h

```diff
@@ -0,0 +1,21 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2021 Intel Corporation. All rights reserved.
+ *
+ * Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+ *
+ */
+
+#ifndef __SOF_MATH_BASE_LOGARITHM_H__
+#define __SOF_MATH_BASE_LOGARITHM_H__
+
+#include <stdint.h>
+
+#define ONE_OVER_LOG2_10_UQ1P43	2647887844335ULL	/* 1/log2(10) */
```

Comment:
Not quite what I asked for, the Q format is in the macro but not clear text. I'd remove the _UQ1P43 from macro (hard to remember when used) and add Q format comment to e.g. /* 1/log2(10), Q1.43 */

Reply:
Sure